### ISSUE TITLE
influxdb-spring-3.0.0.jar has no class com.influxdb.spring.metrics.Influx2MetricsExportAutoConfiguration

### ISSUE THREAD

**Issue by uqix** [influxdata/influxdb-client-java#247]
<img width="696" alt="Screen Shot 2021-07-15 at 4 06 09 PM" src="https://user-images.githubusercontent.com/10142314/125752612-e18ee14d-eea9-4e4b-a304-c80fe29fcf5e.png">

**Comment by bednar** [comment]
Hi @uqix,

thanks for using our client. 

The `micrometer` v1.7.0 brings [support](https://github.com/micrometer-metrics/micrometer/issues/1974) for InfluxDB 2. That is a reason why the `influxdb-spring` no longer needs provide a custom Micrometer metrics exporter. Now you are able to use `micrometer-registry-influx`, for more info [see our docs](./spring/README.md#actuator-for-influxdb2-micrometer-registry). 

Regards

**Comment by uqix** [comment]
Then I guess `spring.factories` in influxdb-spring should not contain `com.influxdb.spring.metrics.Influx2MetricsExportAutoConfiguration`, or we'll get this:

```
java.lang.IllegalStateException: Failed to load ApplicationContext
Caused by: java.lang.IllegalStateException: Unable to read meta-data for class com.influxdb.spring.metrics.Influx2MetricsExportAutoConfiguration
Caused by: java.io.FileNotFoundException: class path resource [com/influxdb/spring/metrics/Influx2MetricsExportAutoConfiguration.class] cannot be opened because it does not exist
```

**Comment by bednar** [comment]
> Then I guess `spring.factories` in influxdb-spring should not contain` com.influxdb.spring.metrics.Influx2MetricsExportAutoConfiguration`, or we'll get this:
>
Yes, you are right, we've missed this. I will prepare PR with fixed version of `influxdb-spring`.

Sorry for troubles.

**Comment by uHan2** [comment]
Hi! By any chance, I'm having the same problem right now, so can you tell me when I can use the solved version?

**Comment by bednar** [comment]
Hi @uHan2, @uqix,

the fixed version `3.0.1` is released: https://repo1.maven.org/maven2/com/influxdb/influxdb-client/.

Regards

**Comment by uHan2** [comment]
@bednar Thank you very much :)
